tools/run_tests/run_tests.py

```diff
@@ -438,16 +456,21 @@ def makefile_name(self):
   def dockerfile_dir(self):
     return 'tools/dockerfile/test/python_jessie_%s' % _docker_arch_suffix(self.args.arch)
 
-  def _get_tox_envs(self, compiler):
-    """Returns name of tox environment based on selected compiler."""
-    if compiler == 'default':
-      return ('py27', 'py34')
-    elif compiler == 'python2.7':
-      return ('py27',)
+  def _get_pythons(self, compiler):
+    if os.name == 'nt':
+      venv_relative_python = 'Scripts/python.exe'
+      toolchain = 'mingw32'
+    else:
+      venv_relative_python = 'bin/python'
+      toolchain = 'unix'
+    python27_config = PythonConfig('python2.7', 'py27', venv_relative_python, toolchain)
+    python34_config = PythonConfig('python3.4', 'py34', venv_relative_python, toolchain)
+    if compiler == 'python2.7' or compiler == 'default':
```

Comment:
Looks like this might have been the result of some creative merge logic:

(My original Python3 commit)
https://github.com/kpayson64/grpc/blob/1efb6017ec1edb26706895dfe71f4d72ea387cf4/tools/run_tests/run_tests.py#L428

Can you change python2.7 to only run python2.7?